bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -1335,4 +1341,62 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
         }
     }
 
+    /**
+     * The method checks configuration of module if it is available and if all required properties are set.
+     * If some property is tries to replace it with its default value.
+     *
+     * @param module checked module
+     */
+    private void checkConfigurations(Module module) {
```

Comment:
from a method called "check", I would not expect that it modifies the passed parameter. Maybe it should be renamed to something like "setDefaults"?